test/utils.js

```diff
@@ -117,8 +117,12 @@ const fixtures = {
 		return accessToken
 	}, api(){
 		return request(app)
+	},
+	generateInviteToken (email){
```

Comment:
i think using the `generateAccessToken` is fine, its not so specific and any parameters needed can be passed to it

we can change the name to just `generateToken` though